From: Lennart Poettering <lennart@poettering.net>
Date: Wed, 8 Apr 2020 16:10:38 +0200
Subject: units: drop dependency on systemd-remount-fs.service from
 systemd-pstore.service

This dependency is now generated automatically given we use
StateDirectory=. Moreover the combination of Wants= and After= was too
strong anyway, as whether remount-fs is pulled in or not should not be up
to systemd-pstore.service, and in fact is part of the initial
transaction anyway.

(cherry picked from commit 0c978faa16fa9ecf92f0bbb5c7cc709dc472d115)
---
 units/systemd-pstore.service.in | 2 --
 1 file changed, 2 deletions(-)

diff --git a/units/systemd-pstore.service.in b/units/systemd-pstore.service.in
index 37fcf87..f9705b5 100644
--- a/units/systemd-pstore.service.in
+++ b/units/systemd-pstore.service.in
@@ -13,8 +13,6 @@ Documentation=man:systemd-pstore(8)
 ConditionDirectoryNotEmpty=/sys/fs/pstore
 ConditionVirtualization=!container
 DefaultDependencies=no
-Wants=systemd-remount-fs.service
-After=systemd-remount-fs.service
 Before=sysinit.target
 
 [Service]
